templates/CRM/Admin/Page/ContactType.tpl

```diff
@@ -66,4 +68,5 @@
     {crmButton q="action=add&reset=1" icon="plus-circle"}{ts}Add Contact Type{/ts}{/crmButton}
     {crmButton p="civicrm/admin" q="reset=1" class="cancel" icon="times"}{ts}Done{/ts}{/crmButton}
   </div>
+</div>
 {/if}
```

Comment:
It looks to me like the opening of the div tag is outside the {if} clause but the closing </div> tag is inside the {/if}

Reply:
This was looking like it due to malformed spacing. I fixed it. Now it should look good.